src/DataViews/PresetsDataView.cpp

```diff
@@ -199,6 +201,13 @@ void PresetsDataView::OnContextMenu(const std::string& action, int menu_index,
                           absl::StrFormat("Could not delete preset \"%s\".", filename));
     }
 
+  } else if (action == kMenuActionShowInExplorer) {
+    if (item_indices.size() != 1) {
```

Comment:
So, if `item_indices.size() == 2`, we will show the option but it won't do anything after clicking it?

Reply:
The UI enforced this to be a single selection already. So we won't have the case that `item_indices.size() == 2`. We have a comment in `GetContextMenuWithGrouping`. I will also add a comment here.

Reply:
Not sure if I got it 100%, if that was enforced in other part, and we are sure that it couldn't be greater than 1, I would write instead:
`if (item_indices.size() == 0) return;`

But if also it shouldn't be 0 because it always should be a single selection (and that is enforced somewhere else), I would rather CHECK it.
`CHECK (item_indices.size() == 1);`

Reply:
Done! Thanks!